internal/integrationtest/board/board_test.go

```diff
@@ -355,8 +355,8 @@ func TestBoardDetails(t *testing.T) {
 		]
 	}`)
 
-	// Download samd core pinned to 1.8.8
-	_, _, err = cli.Run("core", "install", "arduino:samd@1.8.8")
+	// Download samd core pinned to 1.8.10
+	_, _, err = cli.Run("core", "install", "arduino:samd@1.8.10")
```

Comment:
```suggestion
	// Download samd core pinned to 1.8.13
	_, _, err = cli.Run("core", "install", "arduino:samd@1.8.13")
```
Since we are bumping it, why not use the latest one?

Reply:
:bulb: Actually, this test can now be merged with the previous one, as we can directly assert the existence of `debugging_supported` key if we install the `1.8.13` instead of installing the `1.8.6`.

Done in https://github.com/arduino/arduino-cli/pull/2421/commits/2f43e25565c91de3cd4f65d3bfd122face8313c5